Packs/AWSFeed/README.md

```diff
@@ -0,0 +1 @@
+Feed integration for AWS.
```

Comment:
not detailed enough

Reply:
What should be here anyway?

Reply:
Use the readme of the integration itself in the meantime